src/main/java/frc/robot/subsystems/CanalSubsystem.java

```diff
@@ -21,32 +22,44 @@ public class CanalSubsystem extends SubsystemBase {
       new SparkFlex(CanalConstants.CANAL_MOTOR, MotorType.kBrushless);
 
   private final LaserCan laserCan = new LaserCan(5);
+  private LaserCan.Measurement distance = laserCan.getMeasurement();
```

Comment:
```suggestion
  private LaserCan.Measurement distance;
```

Let periodic handle settings this.